--- xemacs-vm-pkg-7.07/man/vm/vm.texinfo~	Tue May 28 20:45:36 2002
+++ xemacs-vm-pkg-7.07/man/vm/vm.texinfo	Sun Jun 30 17:20:45 2002
@@ -1,9 +1,14 @@
 \input texinfo  @comment -*-Text-*-
-@setfilename vm.info
+@setfilename ../../info/vm.info
 @settitle VM User's Manual
+@ifinfo
+@format
+@dircategory XEmacs Editor:
 @direntry
-* VM::			A mail reader.
+* VM: (vm).				A mail reader
 @end direntry
+@end format
+@end ifinfo
 @iftex
 @finalout
 @end iftex
